scaletest/lib/coder_sshtraffic.sh

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env bash
```

Comment:
why is this a bash script instead of part of the scaletest CLI suite?

In addition to bash being gross...

1. We can't get logs or errors
2. We can't generate statistics about latency or bandwidth

Reply:
Primarily time constraints - I wanted to avoid extra code changes.

The lack of logs / errors / latency / bandwidth statistics is definitely an issue.

Reply:
I really think we should stick with go and stick with the scaletest framework, even if it takes longer.

Having ad-hoc traffic generation outside the main framework is going to turn into a big maintenance burden

Reply:
> Primarily time constraints - I wanted to avoid extra code changes.

> I really think we should stick with go and stick with the scaletest framework, even if it takes longer.

If we can achieve the scale testing goal just by using this script, I suppose that it is fine for now. Are there any tests scheduled for the next couple of days? Maybe you should use this script, and iterate on the Go app later?

Reply:
That's my plan currently. I can at least observe the gross effects of SSH traffic for now, and perform more fine-grained testing afterwards.